### DISCUSSION TITLE
Suggests adding scoreboard and sorting ranks on the tablist

### DISCUSSION THREAD

**Discussion by KalviPL** [Zrips/CMI#7774] [Feature Requests]
Suggests adding scoreboard and sorting ranks on the tablist
e.g., some permission

I guess that you would have to rework probably the tab, but I think that in such a plugin should be something like this because we pay for it, and yet it is worthwhile to improve older modules and not just add new ones. Why should server owners add a new plugin for TAB when CMI can have something like that.

**Comment by CloudeLecaw** [comment]
might take a look at the discussions before
you will see and find that there are allready xx other discussion suggestions made by other users.. may enjoy thier disccussions it worth more then every user would create new suggestions there

Request become a bit worthless if each time someone creates a new discussion for a request someone else made before.
They can become worthy and more een if more people discuss in allready existing once :-)
![Snag_36a328e](https://user-images.githubusercontent.com/12080017/232308793-20456a03-965c-4250-a5e0-b7d305e3021e.png)


![Snag_36c38ed](https://user-images.githubusercontent.com/12080017/232308900-667cd20f-a391-4526-a024-80f8af4c2f5e.png)



here a little collection where players who ask for tablist grouping of any kind as a new feature of cmi.

https://github.com/Zrips/CMI/discussions/470

https://github.com/Zrips/CMI/discussions/653

https://github.com/Zrips/CMI/discussions/1480

https://github.com/Zrips/CMI/discussions/1492

https://github.com/Zrips/CMI/discussions/2474

https://github.com/Zrips/CMI/discussions/2671

https://github.com/Zrips/CMI/discussions/3479

https://github.com/Zrips/CMI/discussions/3571

https://github.com/Zrips/CMI/discussions/4197

https://github.com/Zrips/CMI/discussions/4695

https://github.com/Zrips/CMI/discussions/5233

https://github.com/Zrips/CMI/discussions/5376

https://github.com/Zrips/CMI/discussions/5769

https://github.com/Zrips/CMI/discussions/5948

https://github.com/Zrips/CMI/discussions/6031

https://github.com/Zrips/CMI/discussions/6388


https://github.com/Zrips/CMI/discussions/7695

ohh and now
 #7774 

for the Scoreboard please look by yourself :§

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending or descending by type, and types range from permission node weight to baltop rank, etc.
